File: frontend-angular/src/app/core/models/libro.ts
export class Libro {
  id: number;
  nombre: string;
  autor: string;
  edicion: string;
  formato: string;
  tema: string;
  isbn: string;
  precio: number;
  cantidad: number;
  imgname: string;

  constructor() {
    this.id = 0;
    this.nombre = '';
    this.autor = '';
    this.edicion = '';
    this.formato = '';
    this.tema = '';
    this.isbn = '';
    this.precio = 0;
    this.cantidad = 0;
    this.imgname = '';
  }
}

File: backend-csharp/PruebaLibreria/Models/Libro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PruebaLibreria.Models
{
    public class Libro
    {
        public int id {  get; set; }
        public string nombre { get; set; }
        public string autor {  get; set; }
        public string edicion {  get; set; }
        public string formato { get; set; }
        public string tema {  get; set; }
        public float precio {  get; set; }
        public string isbn { get; set; }
        public int cantidad { get; set; }
        public string imgname { get; set; }


    }
}